lib/Reconciler.lua

```diff
@@ -140,6 +100,22 @@ end
 	component tree.
 ]]
 function Reconciler.mount(element, parent, key)
+	if type(element) ~= "table" then
+		if type(element) == "boolean" then
+			-- Ignore booleans of either value
+			-- See https://github.com/Roblox/roact/issues/14
+			return nil
+		end
```

Comment:
Pretty sure moving this check here will break lots of code, since everything in the reconciler uses `_mountInternal`, not `mount.`